index.html

```diff
@@ -223,6 +223,38 @@ <h4 class="uppercase mb16">Community Connect</h4>
 	</div>
 </section>
 
+<!-- Social Embed -->
+<section class="image-bg overlay parallax">
+    <div class="background-image-holder">
+        <img alt="Background Image" class="background-image" src="img/cover11.jpg" />
+    </div>
+    <div class="container">
+        <div class="row">
+            <div class="col-sm-12 text-center">
+                <h3 class="uppercase mb64 mb-xs-32">FOSSASIA in Social Media</h3>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-sm-4">
+                <div class="feature feature-1 boxed">
+                    <a class="twitter-timeline" data-width="400" data-height="400" data-link-color="#E81C4F" href="//twitter.com/fossasia">Tweets by fossasia</a>
+                </div>
+            </div>
+            <div class="col-sm-4">
+                <div class="feature feature-1 boxed">                 
+					<div class="g-page" data-href="//plus.google.com/108920596016838318216" data-rel="publisher"></div>
```

Comment:
Indentation

Reply:
@anshumanv sir, but I think it's indebted. Can you please help me with the changes to make?

Reply:
Sir can you please help me  with the changes  to make?

Reply:
I was offline, so couldn't respond